src/ort_genai_c.cpp

```diff
@@ -246,6 +276,63 @@ OgaResult* OGA_API_CALL OgaTokenizerStreamDecode(OgaTokenizerStream* p, int32_t
   OGA_CATCH
 }
 
+OgaResult* OGA_API_CALL OgaCreateStrings(OgaStrings** out) {
+  OGA_TRY
+  *out = reinterpret_cast<OgaStrings*>(std::make_unique<std::vector<std::string>>().release());
+  return nullptr;
+  OGA_CATCH
+}
+
+OgaResult* OGA_API_CALL OgaCreateAllocatedStrings(size_t num_strings, size_t* str_lengths, OgaStrings** out) {
+  OGA_TRY
+  auto strings = std::make_unique<std::vector<std::string>>(num_strings);
+  for (size_t i = 0; i < num_strings; i++) {
+    strings->at(i).resize(str_lengths[i]);
```

Comment:
Because it is a bounded loop, at() is not necessary

Reply:
Removed OgaStrings type.